src/app/campaign-details/campaign-details.component.html

```diff
@@ -37,17 +37,22 @@ <h1 class="c-header__campaign-title b-rh-1 b-bold">{{ campaign.title }}</h1>
           Campaign preview
         </mat-card-title>
 
-        <mat-card-title *ngIf="campaignInFuture && !isPending">
+        <mat-card-title *ngIf="isDraft">
+          <mat-icon aria-hidden="false" aria-label="Draft campaign">preview</mat-icon>
+          Campaign draft
```

Comment:
I don't think we need to distinguish the scenarios for donors – if they've got to the page we can just provisionally say it 'opens soon'